src/scikit_package/cli/create.py

```diff
@@ -20,3 +83,30 @@ def package(args):
         cookie.run(f"{SKPKG_GITHUB_URL}-conda-forge")
     elif subcmd == "manuscript":
         cookie.run(f"{SKPKG_GITHUB_URL}-manuscript")
+    items_after_run = [f for f in Path().cwd().iterdir()]
+    if args.update:
+        print("Create the package in the update mode.")
+        generated_dirs = list(set(items_after_run) - set(items_before_run))
+        if len(generated_dirs) == 1:
+            project_dir = generated_dirs[0]
+        else:
+            print(
+                "Reverting to the normal mode "
+                "because scikit-package was unable to identify the generated "
```

Comment:
see below.  We need an error message that is useful to the user.  We don't want github issues for this.